setup.py

```diff
@@ -26,88 +23,9 @@ def read(filename):
 
 
 setup(
-    name="zest.releaser",
-    version=version,
-    description="Software releasing made easy and repeatable",
     long_description=long_description,
-    classifiers=[
-        "Development Status :: 6 - Mature",
-        "Intended Audience :: Developers",
-        "License :: OSI Approved :: GNU General Public License (GPL)",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: Implementation :: CPython",
-        "Programming Language :: Python :: Implementation :: PyPy",
-        "Topic :: Software Development :: Libraries :: Python Modules",
-    ],
-    keywords=["releasing", "packaging", "pypi"],
-    author="Reinout van Rees",
-    author_email="reinout@vanrees.org",
-    url="https://zestreleaser.readthedocs.io",
-    license="GPL",
     packages=find_packages(exclude=["ez_setup"]),
```

Comment:
We can probably zap `ez_setup` altogether? Wasn't that sort of a setuptools bootstrap file for when setuptools was new? I don't totally remember :-)

Reply:
Right, it only makes sense to exclude `ez_setup.py` if we actually have this file, which we don't. I removed the code.